setup.py

```diff
@@ -1,41 +1,13 @@
 #!/usr/bin/env python
 
 import os
-from setuptools import setup, find_packages
+from setuptools import setup
 from distutils.core import Extension
 import platform
 
-work_dir = os.path.dirname(os.path.realpath(__file__))
-mod_dir = os.path.join(work_dir, 'src', 'confluent_kafka')
+mod_dir = os.path.join('src', 'confluent_kafka')
```

Comment:
Better to keep it as before if calling it from a different folder

```suggestion
work_dir = os.path.dirname(os.path.realpath(__file__))
mod_dir = os.path.join(work_dir, 'src', 'confluent_kafka')
```